src/data/moves_info.h

```diff
@@ -444,7 +430,7 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .power = 0,
         .type = TYPE_NORMAL,
         .accuracy = 0,
-        .pp = B_UPDATED_MOVE_DATA >= GEN_6 ? 20 : 30,
+        .pp = CONFIG_BY_GEN(30, GEN_6, 20),
```

Comment:
I don't like this usage of the metaprogram to replace existing ternaries.

Reply:
Why? The reason I put them in is for consistency with other fields that have three different values across gens. It would be weird for some to be ternaries and others the CONFIG macro. It also lets you easily expand it.

Reply:
Mainly the reason I posted below
> At a glance, it's not very clear to me what number corresponds to what Gen.